chimney/.jvm/src/test/scala/io/scalaland/chimney/WixSpec.scala

```diff
@@ -260,8 +260,8 @@ object WixSpec extends TestSuite {
       "use placeholder if id is None (IdGeneration.Auto)" - {
```

Comment:
relates to "support SDL @id annotation" in general- should we have a test that shows what happens if it's Some?

Reply:
agree